__init__.py

```diff
@@ -169,13 +176,23 @@ def get_timezone(self, locale):
         """Get the timezone.
 
         This uses a variety of approaches to determine the intended timezone.
+        If locale is the user defined locale, we save that timezone and cache it.
         """
+        
+        # default timezone exisits, so return it.
+        if self.default_timezone:
```

Comment:
Hey, great catch on this! 

I like it overall, there's just an issue with the initial test. This would always return the default timezone after it has been set eg:
"Hey Mycroft, what time is it?" > Report local time
"Hey Mycroft, what time is it in Paris" > Reports local time again

Perhaps something like:
`if self.default_timezone and locale == str(self.default_timezone):`

Also a small typo in the line above :slightly_smiling_face:

Reply:
Hi Kris, good catch, your right!

I changed your line to 
```
if self.default_timezone and locale == self.location_timezone:
```
because the default location will include a locale like "Europe/Paris" and not necessarily the timezone string, right? (At least there is a possibility it is different. "self.default_timezone" will be assigned to "timezone" and not "locale" in line 195)

Hope you are ok with it.

Reply:
Yep you're spot on!

Merging.